drivers/scheduler/k8s/specs/wordpress/wordpress.yaml

```diff
@@ -48,7 +46,46 @@ spec:
         volumeMounts:
         - name: wordpress-persistent-storage
           mountPath: /var/www/html
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 80
+          initialDelaySeconds: 15
+          timeoutSeconds: 5
+        resources:
+          requests:
+            memory: "2Gi"
       volumes:
       - name: wordpress-persistent-storage
         persistentVolumeClaim:
           claimName: wp-pv-claim
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: wordpress
+  labels:
+    app: wordpress
+spec:
+  ports:
+    - port: 80
+      name: web
+  selector:
+    app: wordpress
+    tier: frontend
+  type: NodePort
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
```

Comment:
we can skip the ingress for torpedo since we never use it for any tests. Do any of the client pods talk to ingress?